bench.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/bench');
```

Comment:
Could we do the same thing as in ocs-lite? Meaning copying the `package.json` in the `dist` folder in order to avoid having these files simply re-exporting the entry points?